dhis-2/dhis-services/dhis-service-acl/pom.xml

```diff
@@ -27,15 +27,7 @@
     </dependency>
     <dependency>
       <groupId>org.hisp.dhis</groupId>
-      <artifactId>dhis-support-hibernate</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.hisp.dhis</groupId>
```

Comment:
I agree, we need to add test dependencies somehow.
Maybe for tests I would make it different and use, in a consciousness way, transitive dependencies.
So it could make sense to have dhis-support-test for any module that has tests and do not declare in the pom module any test related dependency